index.html

```diff
@@ -0,0 +1,104 @@
+        <div class="ham-bar"></div>
+        <div class="ham-bar"></div>
+      </div>
+      <div class="menu">
+        <div class="menu-element close">
+          <i class="fa fa-times fa-1x"></i>
+        </div>
+        <ul class="nav-list">
+          <li><a href="#home" class="menu-element loaded">Home</a></li>
+          <li><a href="about.html" class="menu-element">About</a></li>
+          <li><a href="#main-program" class="menu-element">Program</a></li>
+          <li><a href="#main-program" class="menu-element">Sponsors</a></li>
+          <li><a href="#main-program" class="menu-element">News</a></li>
+          <li><a href="#main-program" class="menu-element" id="campaign">Campaign</a></li>
+        </ul>
+      </div>
+    </nav>
+    <div class="intro">
+      <span>"Hello! Young Techie"</span>
+      <h1 id="hero-text">
+        Project Managers <br> Global Workshop <br> 2022
+      </h1>
+    </div>
+    <p id="keynote">
+      Novustack is beginning the second quarter of 2022 with a fantastic offer for young techies looking to get their 
+      foot in the door of Product Management. 
+    </p>
+    <p class="event-date">2022.04.19(TUE) ~ 21(THU)</p>
+    <p class="event-venue">@ 50 Ebitu Ukiwe St, Jabi 900108, Abuja, Nigeria</p>
+  </header>
+  <main>
+    <section id="main-program">
+
+      <h2>Main Program</h2>
+      <hr class="heading-underline">
+      <div class="program-cards-container">
+        <div class="program-card">
+          <img src="./assets/img/lecture-icon.png" alt="Lecture Icon">
+          <h2 class="card-title">Lecture</h2>
+          <p>Listen to, among other speakers, a Harvard and MIT graduate with expertise as a former 
+            Amazon Technical Product Manager</p>
+        </div>
+        <div class="program-card">
+          <img src="./assets/img/exhition-icon.png" alt="Exhibition Icon">
+          <h2 class="card-title">PM Displays</h2>
+          <p>Appreciate various products made by proper application of project management concepts</p>
+        </div>
+        <div class="program-card">
+          <img src="./assets/img/forum-icon.png" alt="Forum Icon">
+          <h2 class="card-title">Forum</h2>
+          <p>Have time to discuss your ideas and thoughts with specialists on each subject.</p>
+        </div>
+        <div class="program-card">
+          <img src="./assets/img/workshop-icon.png" alt="Workshop Icon">
+          <h2 class="card-title">Workshop</h2>
+          <p>Try creating your own using open source license rather than just watching</p>
+        </div>
+        <div class="program-card">
+          <img src="./assets/img/network-icon.png" alt="Network Icon">
+          <h2 class="card-title">PM Ignite</h2>
+          <p>Get opportunies to network with Novustack affiliate around the world, also after the summit</p>
+        </div>
+      </div>
+      <button type="button">Join PM Global Workshop 2022</button>
+    </section>
+
+    <section id="speakers"></section>
+
```

Comment:
- [ ] Each article in the speakers section should have only 2 columns (speaker cards) per row. In your case, you have 3 speakers per row as shown below: 👇 
### Your Design
![image](https://user-images.githubusercontent.com/46839548/164912686-e08b6be7-9954-46b7-a09a-1703472d4d0e.png)
### Behance Design
![image](https://user-images.githubusercontent.com/46839548/164912700-102d4fc1-0576-4239-88e5-62f2d617baf5.png)

Reply:
[ ] - Kindly make these section 2 columns as requested by the previous reviewer